fuel-tx/src/transaction/types/storage.rs

```diff
@@ -18,49 +18,38 @@ use rand::{
 
 use core::cmp::Ordering;
 
+pub type StorageData = Vec<u8>;
```

Comment:
I'm curious about the naming of this. Are we intending _all_ storage to move to `Vec` or just for contract state? Or is there no distinction between storage and contract storage?

Just trying to understand the scope of all this.

Reply:
This can be named better - something like `ContractsStateData` could be more congruent with the table, where the key is `ContractsStateKey`. I will make that change, but let me know if you have a suggestion you prefer.

Reply:
I renamed it to `ContractsStateData` to illustrate that this is part of the `ContractsState` table alongside the `ContractsState` key.